Error while making OCL sources on CentOS 9 with kernel 5.14.0-467.el9.x86_64

## Logs
make[2]: Leaving directory '/root/open-cas-linux/casadm'
make[1]: Leaving directory '/root/open-cas-linux/casadm'
  CC [M]  /root/open-cas-linux/modules/cas_cache/classifier.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/context.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/control.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/disk.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/exp_obj.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/layer_cache_management.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/main.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/ocf_env.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/service_ui_ioctl.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/threads.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/utils/utils_data.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/utils/utils_gc.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/utils/utils_mpool.o
  CC [M]  /root/open-cas-linux/modules/cas_cache/utils/utils_rpool.o
/root/open-cas-linux/modules/cas_cache/exp_obj.c:51: error: "KRETURN" redefined [-Werror]
   51 |                 #define KRETURN(x)      return
      |
/root/open-cas-linux/modules/cas_cache/exp_obj.c:38: note: this is the location of the previous definition
   38 |        #define KRETURN(x)      ({ return (x); })
      |
/root/open-cas-linux/modules/cas_cache/exp_obj.c:52: error: "MAKE_RQ_RET_TYPE" redefined [-Werror]
   52 |                 #define MAKE_RQ_RET_TYPE void
      |
/root/open-cas-linux/modules/cas_cache/exp_obj.c:39: note: this is the location of the previous definition
   39 |        #define MAKE_RQ_RET_TYPE blk_qc_t
      |
  CC [M]  /root/open-cas-linux/modules/cas_cache/volume/vol_blk_utils.o
cc1: all warnings being treated as errors
make[4]: *** [scripts/Makefile.build:268: /root/open-cas-linux/modules/cas_cache/exp_obj.o] Error 1
make[4]: *** Waiting for unfinished jobs....
make[3]: *** [scripts/Makefile.build:556: /root/open-cas-linux/modules/cas_cache] Error 2
make[2]: *** [Makefile:1942: /root/open-cas-linux/modules] Error 2
make[2]: Leaving directory '/usr/src/kernels/5.14.0-467.el9.x86_64'
make[1]: *** [Makefile:46: default] Error 2
make[1]: Leaving directory '/root/open-cas-linux/modules'
make: *** [Makefile:22: modules] Error 2


## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* OpenCAS version (commit hash or tag): eb97598e484b89dfecf55a9a39ecb5244aaaab03 (current origin/master)
* Operating System: CentOS 9
* Kernel version: 5.14.0-467.el9.x86_64

--- a/modules/cas_cache/exp_obj.c
+++ b/modules/cas_cache/exp_obj.c
@@ -48,6 +48,8 @@ static inline void bd_release_from_disk(struct block_device *bdev,
 /* For RHEL 9.x we assume backport from kernel 5.18+ */
 #ifdef RHEL_MAJOR
 	#if RHEL_MAJOR >= 9
+		#undef KRETURN
+		#undef MAKE_RQ_RET_TYPE
 		#define KRETURN(x)      return
 		#define MAKE_RQ_RET_TYPE void
 	#endif

